lib/split/dashboard/pagination_helpers.rb

```diff
@@ -3,10 +3,9 @@
 
 module Split
   module DashboardPaginationHelpers
-    DEFAULT_PER = 10
-
     def pagination_per
-      @pagination_per ||= (params[:per] || DEFAULT_PER).to_i
+      default_per = Split.configuration.dashboard_pagination_default_per
+      @pagination_per ||= (params[:per] || default_per).to_i
```

Comment:
Can you also add a spec on [dashboard_spec.rb](https://github.com/splitrb/split/blob/master/spec/dashboard_spec.rb) to ensure we're pulling the default from the configuration?